res/skins/Tango/skin.xml

```diff
@@ -115,6 +115,52 @@
       <attribute persist="false" config_key="[EffectRack1_EffectUnit4],group_[BusTalkover]_enable">0</attribute>
     </attributes>
   </manifest>
+  <!--
+  **********************************************
+  General- Skin Colour Scheme
+  **********************************************
+  -->
+  <Schemes>
+    <Scheme>
+      <Name>Classic</Name>
+      <Filters/>
+      <!-- Colors for waveform & overview -->
+      <SetVariable name="SignalColor_12">#5EBAFF</SetVariable>
+      <SetVariable name="SignalBgColor_12">#001D33</SetVariable>
+      <SetVariable name="SignalColor_34">#DF4DF0</SetVariable>
+      <SetVariable name="SignalBgColor_34">#3A0440</SetVariable>
+      <!-- small Spinny mask and default mini cover should match -->
+      <SetVariable name="SpinnyCoverColor_12">blue</SetVariable>
+      <SetVariable name="SpinnyCoverColor_34">purple</SetVariable>
+      <Style>
+        <!-- All text labels in decks (artist, title, time, BPM, key, ...) -->
+        #Deck1 WLabel,#RateContainer1 WLabel, #RateContainer1 WPushButton,
+        #Deck2 WLabel, #RateContainer2 WLabel, #RateContainer2 WPushButton { color: #33a8ff;}
+        #Deck3 WLabel, #RateContainer3 WLabel, #RateContainer3 WPushButton,
+        #Deck4 WLabel, #RateContainer4 WLabel, #RateContainer4 WPushButton { color: #d910f0;}
+        <!-- Vinyl Controls toggle is transparent. This sets bg color so it matches those above -->
+        #DeckOverview1, #DeckOverview2 { background-color: #001D33;}
+        #DeckOverview3, #DeckOverview4 { background-color: #3A0440;}
```

Comment:
I used inline css for this small change in order to have all scheme colors in one place.